index.html

```diff
@@ -3,9 +3,35 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins&display=swap">
+    <link rel="stylesheet" href="./fonts/CreteRound-Italic.ttf">
     <title>Mobile Portfolio</title>
 </head>
 <body>
```

Comment:
✅ Good job on implementing the background image in the headline section! However, I have a couple of suggestions to improve your design:

1. [Optional] Instead of setting the background image in the body, it would be better to add the image directly to the headline section as per the design. This will make it easier for you to align the background image with the specific section, especially when you add more sections in the future. Consider updating your code to reflect this change.

2. [Optional] In the 375px view, there seems to be an issue with the background image overlapping the paragraph in the headline section. To ensure a consistent and visually appealing layout across different screen sizes, please fix the overlapping by adjusting the positioning or size of the background image.

| Your | Design |
| --- | --- |
| ![image](https://github.com/prince-rutagengwa/mobile-portfolio/assets/55840999/7ebe9a21-90c4-4c6d-8154-7dfebb765ae4) | ![image](https://github.com/prince-rutagengwa/mobile-portfolio/assets/55840999/30146154-999d-4b4d-9ff8-e4650e080206) |

Taking these suggestions into account will help you achieve a design that closely matches the specifications in the design. Keep up the good work! 🎉👍